ksqldb-engine/src/main/java/io/confluent/ksql/schema/ksql/inference/SchemaRegisterInjector.java

```diff
@@ -96,7 +99,8 @@ private void registerForCreateAs(final ConfiguredStatement<? extends CreateAsSel
         queryMetadata.getResultTopic().getKafkaTopicName(),
         queryMetadata.getResultTopic().getValueFormat().getFormatInfo(),
         cas.getConfig(),
-        cas.getStatementText()
+        cas.getStatementText(),
+        true
```

Comment:
Just to make sure, in which case to we want to overwrite existing schemas?

Reply:
we want to overwrite existing schemas in the case of `CREATE * AS SELECT`